src/transactions/TransactionUtils.cpp

```diff
@@ -19,6 +19,38 @@
 namespace stellar
 {
 
+static uint32_t ACCOUNT_SUBENTRY_LIMIT = 1000;
+static size_t MAX_OFFERS_TO_CROSS = 1000;
```

Comment:
Should we put these behind a `#ifdef BUILD_TESTS`, and used `const` versions of these variables otherwise?

Reply:
done